tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -187,7 +187,8 @@ public static void ClearProfilerEnvironmentVariables()
             string processToProfile = null,
             bool enableSecurity = false,
             bool enableBlocking = false,
-            string externalRulesFile = null)
+            string externalRulesFile = null,
+            int? traceRateLimit = null)
```

Comment:
The goal with this method is to set the minimal required env vars to enable automatic tracing (`DD_DOTNET_TRACER_HOME`, `COR*`, etc), and anything else is passed through the  `environmentVariables` parameter. I guess nobody caught this before because this is the 4th parameter that AppSec is adding:
```
enableSecurity
enableBlocking
externalRulesFile
traceRateLimit
```

Can you use `environmentVariables` instead of adding `traceRateLimit`? We can come back and clean up the previous ones later, no need to that in this PR.

Reply:
fixed in 8a1b7d7cfb8567399adc367e4bb2c284b1a35ea7

Reply:
Thanks!